Adding 'harleen' theme

aneveux opened the pull request:
Hello guys!

Thanks a lot for the really good work you guys are doing!

Just wanted to contribute that minimal theme I'm using. Hope that looks interesting for you :)

Do not hesitate to let me know in case any modification should be made.

Have a great day!

aneveux commented:
Hello @sagebind :)

Any news on that PR? Is there anything I could do regarding it?

Thanks a lot anyway for your attention :)

Have a great day!

sagebind commented:
@aneveux Sorry, time sure flies and our team sure has a small amount of it. 😁 

I finished reviewing the theme in the test environment. One nitpick, I would avoid using `fish_greeting.fish` for defining functions or variables used in the rest of the theme as it breaks when users have a custom greeting defined. It also does not work on first install:

```
fish: Unknown command 'blue'
~/.config/fish/functions/fish_prompt.fish (line 1): blue
                                                    ^
in command substitution
        called on line 45 of file ~/.config/fish/functions/fish_prompt.fish

in function 'fish_prompt'
        called on standard input

in command substitution
        called on standard input

fish: Unknown command 'dim'
~/.config/fish/functions/fish_prompt.fish (line 1): dim
                                                    ^
in command substitution
        called on line 45 of file ~/.config/fish/functions/fish_prompt.fish

in function 'fish_prompt'
        called on standard input

in command substitution
        called on standard input

# etc..
```

See also this issue here for more context: https://github.com/oh-my-fish/oh-my-fish/issues/483

aneveux commented:
Hello @sagebind!

Hope you're doing great today :)

No worries about the time, I completely understand you guys have a lot of things to do :) Just wanted to be sure you guys had a notification I submitted something.

Thanks a lot for all those nice and helpful remarks! I actually didn't know much on how to proceed and just chose to take inspiration from some existing themes. Sadly for me I just picked two of the ones impacted by the problems you mentionned (sushi and batman).

I had a look at the issue you mentionned and just rewrote all my theme taking inspiration from the default one this time. I tested it on a fresh installation and it worked really fine on my side. Nothing's left in that greeting file anymore :)

Do not hesitate to have a look in case you have some time: https://github.com/aneveux/theme-harleen

I updated everything including the screenshot.

Thanks a lot again for your time and attention. Have an awesome day, and thanks again for that nice review :)

sagebind commented:
Looks good to me! `harleen` will be installable by name for all users immediately after merging.

aneveux commented:
Thanks a lot @sagebind, thanks a lot @derekstavis :)
